pkg/daemon/daemon.go

```diff
@@ -905,6 +906,12 @@ func tryEnableTun() {
 	}
 }
 
+func tryEnableVhostNet() {
+	if err := utils.LoadKernelModule("vhost-net"); err != nil {
```

Comment:
Maybe better to change to `vhost_net`, as `lsmod` print modules with underscore and we would not grep it in [load-kmod.sh](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/bindata/scripts/load-kmod.sh)


```
lsmod | grep "vhost_net"                                                                                                                                                                                             
vhost_net              28672  0
vhost                  49152  1 vhost_net
tap                    28672  1 vhost_net
tun                    49152  4 vhost_net
```

```suggestion
	if err := utils.LoadKernelModule("vhost_net"); err != nil {
```

Reply:
both of them work I am fine chaining :)